calculatorBackend/src/exe/main.cpp

```diff
@@ -21,9 +21,9 @@ int main()
     bool running = true;
     while(running)
     {
-        calculator::Calculator calculator;
-        calculator.runCalculator(history);
-        running = calculator.restart(history);
+        // calculator::Calculator calculator;
```

Comment:
Make a comment that if uses uncomment this and refactor, they can retrieve history from the calculator